src/main/java/com/spotinst/sdkjava/enums/admin/account/UserTypeEnum.java

```diff
@@ -0,0 +1,38 @@
+package com.spotinst.sdkjava.enums.admin.account;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public enum UserTypeEnum {
```

Comment:
Which API is using this enum?

Reply:
UserDetails.java and OrganizationUsers.java are using this enum